docker/Dockerfile-osc-base

```diff
@@ -2,24 +2,42 @@ FROM centos:latest
 
 MAINTAINER Arvind Nadendla <arvindn05@gmail.com>
 
-COPY proxy.sh /etc/profile.d/
-
-RUN source /etc/profile.d/proxy.sh && \
-yum update -y && \
+RUN yum update -y && \
 yum install -y wget && \
 wget --no-check-certificate --no-cookies --header "Cookie: oraclelicense=accept-securebackup-cookie" http://download.oracle.com/otn-pub/java/jdk/8u162-b12/0da788060d494f5095bf8624735fa2f1/jre-8u162-linux-x64.rpm && \
 yum localinstall -y jre-8u162-linux-x64.rpm && \
 rm -rf jre-8u162-linux-x64.rpm && \
+wget --no-check-certificate --no-cookies --header "Cookie: oraclelicense=accept-securebackup-cookie" http://download.oracle.com/otn-pub/java/jdk/8u162-b12/0da788060d494f5095bf8624735fa2f1/jdk-8u162-linux-x64.rpm && \
```

Comment:
oracle JDK cannot be distributed as a docker image. Please use JRE only.

Reply:
@arvindn05 maven command do not build OSC if we use jre, I think we can remove the jdk once the OSC is built.

Reply:
this puts JDK into the docker image file. we cannot have that. Compiling of OSC happens outside of docker so we should not required JDK within the docker.

Reply:
ok. can we use multi stage build for building OSC? 

If you are not using multi stage build, we dont need to build osc in the docker image.

also travis is failing.

Reply:
Now I have removed the OSC build part in docker image.  
I thought of making multi stage build. Problems with multistage build are travis generates the log exceeding 4Mb which causes travis build to fail, its a limitation in travis, need to make extra work around for it and also the kolla uses docker version  that does not support multi stage build.  The reason for travis build failure was due to exceeding log output since OSC is built two times once for jacoco analysis and another inside docker image.